<?php

return [
    'Register New Species' => 'Register New Species',
    'About Us' => 'About Us',
  	'New Language Set Successfully' => 'New Language Set Successfully',
    'Who_developed_text' => 'The RGeasy tool has originated from a partnership between researchers of the Laboratory of Molecular Analysis (LAM) and the Computer Science Course from the Federal University of Tocantins (UFT), along with researchers of the Plant Molecular Physiology Laboratory from the Federal University of Lavras (UFLA).',
    'Who_developed' => 'Who developed the RGeasy tool?',
    'What_does_it_do' => 'What does the RGeasy tool do?',
    'What_does_it_do_text' => 'RGeasy helps to select reference genes for quantitative gene expression analysis by RT-qPCR in an easier way, since information such as Cq\'s values, usually required by other tools, is not necessary at the moment of the search, because information such as these are previously made available by the researchers developing the studies of reference gene validation, enabling the classification of reference genes for the combination of different conditions or treatments, which is not often addressed in most studies. RGeasy uses the RefFinder algorithm (XIE et al. 2012) for the classification of reference genes, resulting in a ranking of the most stable genes, and it still provides sets of primers for the selected genes.',
    'References' => 'REFERENCES',
    'References_text' => 'XIE, Fuliang et al. miRDeepFinder: a miRNA analysis tool for deep sequencing of plant small RNAs. Plant molecular biology, v. 80, n. 1, p. 75-84, 2012.',
    'edit articles' => 'Edit Articles',
    'activate articles' => 'Activate Articles',
    'edit species' => 'Edit Species',
    'activate species' => 'Activate Species'
];
